wrapping/itkRLERegionOfInterestImageFilter.wrap

```diff
@@ -0,0 +1,3 @@
+itk_wrap_class("itk::RLERegionOfInterestImageFilter" POINTER)
```

Comment:
The file is called `RLERegionOfInterestImageFilter`, but the class is called `RegionOfInterestImageFilter` - no RLE prefix, it is just a specialization of regular `RegionOfInterestImageFilter`.

Reply:
👍 Changed in e114d83.

I wonder whether two classes having the same name is correct can lead to some confusion, though.

Reply:
**RoI** filters must have the same name. `itk::RLEImage` class was designed to be a drop-in replacement for `itk::Image`.